internal/factsengine/gatherers/corosynccmapctl.go

```diff
@@ -0,0 +1,45 @@
+package gatherers
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/trento-project/agent/internal/utils"
+)
+
+const (
+	CorosyncCmapCtlFactKey = "corosync-cmapctl"
```

Comment:
In the package version gatherer I have used `_` as "splitter". We should agree on this hehe
`-` or `_`?

Reply:
it's not really splitting anything, but reflecting the name of the binary as-is in the same way `corosync.conf` is the fact key for the corosync configuration gatherer. I still understand the confusion and don't have a strong opinion on this, wdyt?